zerver/actions/message_send.py

```diff
@@ -1154,7 +1154,7 @@ def check_send_private_message(
 def check_send_message(
     sender: UserProfile,
     client: Client,
-    message_type_name: str,
+    message_type: str,
```

Comment:
I think `recipient_type_name` would be a better variable name, since it's really a type of recipient, and `recipient_type` might refer to the 1/2/3 constants for `STREAM`, `HUDDLE`, etc.

Reply:
Updated the prep commit to use `recipient_type_name` and reviewed a `git-grep` of what's remaining in `/zerver` for `message_type`. They are all either part of the events change that we're not planning for this release or are unrelated to this part of the codebase.

```console
$ git grep -c message_type zerver/
zerver/actions/invites.py:1
zerver/actions/message_delete.py:5
zerver/actions/message_edit.py:1
zerver/actions/message_send.py:2
zerver/actions/typing.py:2
zerver/lib/event_schema.py:11
zerver/lib/events.py:1
zerver/migrations/0377_message_edit_history_format.py:3
zerver/openapi/zulip.yaml:14
zerver/tests/test_events.py:4
zerver/tests/test_realm_export.py:1
zerver/tests/test_retention.py:1
zerver/tests/test_service_bot_system.py:2
zerver/views/realm_export.py:1
zerver/webhooks/groove/fixtures/ticket_assigned__agent_and_group.json:1
zerver/webhooks/groove/fixtures/ticket_assigned__agent_only.json:1
zerver/webhooks/groove/fixtures/ticket_assigned__group_only.json:1
zerver/webhooks/groove/fixtures/ticket_assigned__no_one.json:1
zerver/webhooks/groove/fixtures/ticket_started.json:1
zerver/webhooks/groove/fixtures/ticket_state_changed.json:1
zerver/webhooks/pagerduty/view.py:10

```